tests/test_account.py

```diff
@@ -1057,6 +1057,31 @@ def test_set_emr_block_public_access_configuration(self):
         self.assertEqual(resp["BlockPublicAccessConfiguration"]
             ["PermittedPublicSecurityGroupRuleRanges"][0]['MaxRange'], 23)
 
+    def test_ses_agg_send_stats(self):
+        factory = self.replay_flight_data('test_ses_agg_send_stats')
+        p = self.load_policy({
+            'name': 'ses-agg-send-stats-policy',
+            'resource': 'account',
+            'filters': [{"type": "ses-agg-send-stats"}]},
+            config={'region': 'us-west-2'},
+            session_factory=factory)
+        resources = p.run()
+        self.assertEqual(len(resources), 1)
+
+    def test_ses_consecutive_send_stats(self):
+        factory = self.replay_flight_data('test_ses_agg_send_stats')
+        p = self.load_policy({
+            'name': 'ses-consecutive-stats',
+            'resource': 'account',
+            'filters': [{"type": "ses-send-stats", "days": 2}]},
+            config={'region': 'us-west-2'},
+            session_factory=factory)
+        with mock_datetime_now(parser.parse("2022-10-26T00:00:00+00:00"), datetime):
+            resources = p.run()
+        self.assertEqual(len(resources), 1)
+        for r in resources:
+            self.assertEqual(r['c7n:ses-send-stats'][0]['Date'], "2022-10-25")
```

Comment:
@kk1532 Still not enough assertions. What about the other date, is it going to be `2022-10-26` or `2022-10-24`? Looking at the logic of the code, it's actually going to be `2022-10-24`.  So we should be updating the filter description to say "This filter annotates the account resource with SES send statistics for the last n number of days, not including the current date."

Additionally, the test is not verifying that we're adding things up correctly. If someone modifies the code and introduces a bug with adding or dividing or rounding things up, etc, this test wouldn't catch it. Here's how I think the tests should be
```
    def test_ses_agg_send_stats(self):
        factory = self.replay_flight_data('test_ses_agg_send_stats')
        p = self.load_policy({
            'name': 'ses-agg-send-stats-policy',
            'resource': 'account',
            'filters': [{"type": "ses-agg-send-stats"}]},
            config={'region': 'us-west-2'},
            session_factory=factory)
        resources = p.run()
        expected_agg_send_stats = {
            'DeliveryAttempts': 130, 'Bounces': 1, 'Complaints': 0,
            'Rejects': 0, 'BounceRate': 1
        }
        self.assertEqual(len(resources), 1)
        self.assertEqual(resources[0]['c7n:ses-send-agg'], expected_agg_send_stats)

    def test_ses_consecutive_send_stats(self):
        factory = self.replay_flight_data('test_ses_agg_send_stats')
        p = self.load_policy({
            'name': 'ses-consecutive-stats',
            'resource': 'account',
            'filters': [{"type": "ses-send-stats", "days": 2}]},
            config={'region': 'us-west-2'},
            session_factory=factory)
        with mock_datetime_now(parser.parse("2022-10-26T11:00:00+00:00"), datetime):
            resources = p.run()

        self.assertEqual(len(resources), 1)
        expected_send_stats = [
            {
                'DeliveryAttempts': 17, 'Bounces': 1, 'Complaints': 0,
                'Rejects': 0, 'BounceRate': 6, 'Date': '2022-10-25'
            },
            {
                'DeliveryAttempts': 20, 'Bounces': 0, 'Complaints': 0,
                'Rejects': 0, 'BounceRate': 0.0, 'Date': '2022-10-24'
            }
        ]
        self.assertEqual(resources[0]['c7n:ses-send-stats'], expected_send_stats)
        self.assertEqual(resources[0]['c7n:ses-max-bounce-rate'], 6)
```

Reply:
suggestions are addressed.